docs/source/en/using-diffusers/loading_adapters.md

```diff
@@ -157,6 +157,22 @@ image
 
 For both [`~loaders.LoraLoaderMixin.load_lora_weights`] and [`~loaders.UNet2DConditionLoadersMixin.load_attn_procs`], you can pass the `cross_attention_kwargs={"scale": 0.5}` parameter to adjust how much of the LoRA weights to use. A value of `0` is the same as only using the base model weights, and a value of `1` is equivalent to using the fully finetuned LoRA.
 
+For fine-grained control on how much of the LoRA weights are used, use [`~loaders.LoraLoaderMixin.set_adapters`]. Here, you can define scale of any granularity up to per-transformer.
+```python
+pipe = ... # create pipeline
+pipe.load_lora_weights(..., adapter_name="my_adapter") 
+scales = {
+    "text_encoder": 0.5,
+    "text_encoder_2": 0.5,  # only usable if pipe has a 2nd text encoder
+    "down": 0.9,  # all transformers in the down-part will use scale 0.9
+    # "mid"  # because "mid" is not given, all transformers in the mid part will use the default scale 1.0
+    "up": {
+        "block_0": 0.6,  # all 3 transformers in the 0th block in the up-part will use scale 0.6
+        "block_1": [0.4, 0.8, 1.0],  # the 3 transformers in the 1st block in the up-part will use scales 0.4, 0.8 and 1.0 respectively
+    }
+}
+pipe.load_lora_weights("my_adapter", scales)
+```
```

Comment:
Since this is more substantial now, I think it'd be better to create a new subsection. Something like:

```md
### Adjust LoRA weight scale

For both load_lora_weights() and load_attn_procs()...

new content about using `scales`
```